collect_app/src/main/java/org/odk/collect/android/preferences/PreferencesActivity.java

```diff
@@ -45,285 +41,249 @@
 import org.odk.collect.android.logic.PropertyManager;
 import org.odk.collect.android.utilities.MediaUtils;
 import static org.odk.collect.android.preferences.PreferenceKeys.*;
+import static org.odk.collect.android.preferences.AdminAndGeneralKeys.ag;
 
 import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Handles general preferences.
  *
  * @author Thomas Smyth, Sassafras Tech Collective (tom@sassafrastech.com;
  *         constraint behavior option)
  */
-public class PreferencesActivity extends PreferenceActivity implements OnPreferenceChangeListener {
-
+public class PreferencesActivity extends AppPreferenceActivity implements OnPreferenceChangeListener {
+    private final String TAG = getClass().getSimpleName();
```

Comment:
I'll add that although it's currently off, the intent is to turn it back on ASAP so we shouldn't use this trick.